src/examples/ContentDetail/contentDetail.js

```diff
@@ -15,27 +15,27 @@ const contentDetail = document.getElementById('contentDetail');
 
 let contentItems = [];
 
-function getContent() {
-    contentList.innerHTML = 'Loading ...';
-    decent.content().searchContent(new decent.SearchParams())
-        .then(content => {
-            contentList.innerHTML = renderContent(content);
-            contentItems = content;
-        })
-        .catch(err => {
-            console.error(err);
-            contentList.innerHTML = 'Error loading content';
-        });
-}
+contentList.innerHTML = 'Loading ...';
+decent.content().searchContent(new decent.SearchParams())
+    .then(content => {
+        contentList.innerHTML = renderContent(content);
+        contentItems = content;
+    })
+    .catch(err => {
+        console.error(err);
+        contentList.innerHTML = 'Error loading content';
+    });
 
 function renderContent(content) {
-    let render = '<ul>';
-     render += content.map(c => '<li onclick="showDetail(\'' + c.id + '\')"><a href="#">' + c.synopsis.title + '</a></li>');
-     render += '</ul>';
-     if (content.length === 0) {
-         render = '<h3>No content</h3>';
-     }
-     return render
+    let render = '';
```

Comment:
what about improve readability and performance:

const render = [];
render.push('<h3>No content</h3>');
...
return render.join("");

Reply:
On the way